datahub-graphql-core/src/main/resources/auth.graphql

```diff
@@ -86,4 +130,106 @@ type AccessToken {
   """
   accessToken: String!
 
+  """
+  Metadata about the generated token
+  """
+  metadata: AccessTokenMetadata
+}
+
+"""
+Input arguments for listing access tokens
+"""
+input ListAccessTokenInput {
+  """
+  The starting offset of the result set
+  """
+  start: Int
+
+  """
+  The number of results to be returned
+  """
+  count: Int
+
+  """
+  Facet filters to apply to search results
+  """
+  filters: [FacetFilterInput!]
+}
+
+"""
+Results returned when listing access tokens
+"""
+type ListAccessTokenResult {
+  """
+  The starting offset of the result set
+  """
+  start: Int!
+
+  """
+  The number of results to be returned
+  """
+  count: Int!
+
+  """
+  The total number of results in the result set
+  """
+  total: Int!
+
+  """
+  The token metadata themselves
+  """
+  tokens: [AccessTokenMetadata!]!
+}
+
+type AccessTokenMetadata implements Entity {
+
+  """
+  The primary key of the access token
+  """
+  urn: String!
+
+  """
+  The standard Entity Type
+  """
+  type: EntityType!
+
+  """
+  The unique identifier of the token.
+  """
+  tokenId: String!
```

Comment:
Any reason to call this "tokenId" and not just "id", and same with the name field? Why the prefix? Seems redundant.